bootstrap.py

```diff
@@ -337,8 +337,8 @@ def install_katello_agent():
     """Install Katello agent (aka Gofer) and activate /start it."""
     print_generic("Installing the Katello agent")
     call_yum("install", "katello-agent")
-    exec_failexit("/sbin/chkconfig goferd on")
-    exec_failexit("/sbin/service goferd restart")
+    enable_service("goferd")
+    start_service("restart", "goferd")
```

Comment:
It can't be cause the command parameter has assigned a default value equal to start at definition time. That's why the service which has no default value at definition time must be prior to command otherwise you should expect a python error.